services/explorer/db/consumerinterface.go

```diff
@@ -24,7 +24,7 @@ type ConsumerDBWriter interface {
 	StoreTokenIndex(ctx context.Context, chainID uint32, tokenIndex uint8, tokenAddress string, contractAddress string) error
 	// StoreSwapFee stores the swap fee data.
 	StoreSwapFee(ctx context.Context, chainID uint32, timestamp uint64, contractAddress string, fee uint64, feeType string) error
-	// UNSAFE_DB gets the underlying gorm db. This is not intended for use in production.
+	// UNSAFE_DB gets the underlying gorm db. This is for testing only and not intended for use in production.
```

Comment:
fyi, just added these nits because I wanted to [goreleaser] tag w/o a force push